### DISCUSSION TITLE
Need help with an error

### DISCUSSION THREAD

**Discussion by hadieshraghi6** [catalyst-cooperative/pudl#3594] [Help Me!]
Hello,
I think I successfully activated pudl-catalog environment. I was able to run pudl_cat.hourly_emissions_epacems successfully but when trying to run pudl_cat.hourly_emissions_epacems.discover(), I get this error:

FileNotFoundError: An error occurred while calling the read_parquet method registered to the pandas backend.
Original Message: intake.catalyst.coop/v2022.11.30/hourly_emissions_epacems.parquet

Looks like this .parquest file can't be found. Can you explain what need to be done?

**Comment by zaneselvans** [comment]
Hey there, sorry we've deprecated the Intake catalogs, but the data is all still available. Check out the [Data Access page of the PUDL docs](https://catalystcoop-pudl.readthedocs.io/en/nightly/data_access.html).
